build/yamls/antrea-ipsec.yml

```diff
@@ -490,6 +490,12 @@ spec:
           periodSeconds: 10
           timeoutSeconds: 5
         name: antrea-agent
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 6443
```

Comment:
We don't use 6443, where do you get it?

Reply:
It is the default Kubernetes api-server port and I changed it after test failed when using port 10443. I will switch to port 10443 and try tests again.